lib/app.js

```diff
@@ -94,6 +95,14 @@
 				passes = passes.concat(report.passes);
 				failures = failures.concat(report.failures);
 
+				if(options.ignore || lintOptions.ignore) {
+					var ignoreOptions = {ignore:options.ignore};
+					ignoreOptions.ignore = merge(ignoreOptions.ignore, lintOptions.ignore);
```

Comment:
I don't like whats going on here.
The actual linter should be updated to expect a more normal looking onject